### ISSUE TITLE
Make the home page pretty

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1787]
### Links of affected pages in Plone Documentation, if any.

https://6.docs.plone.org/

### Description

The home page is not great. For inspiration, see:

- https://numpy.org/doc/stable/
- https://pydata-sphinx-theme.readthedocs.io/en/stable/

We can use [Sphinx Design](https://sphinx-design.readthedocs.io/en/latest/) to have a card-based layout.

We currently have no User Guide. See #1611 for the plan.

We have solid "Contributing guide" and "Get started", and the beginning of both "Developer guide" and "Conceptual guides".

We need a new page for "Reference guide" that includes links to any reference material, including plone.api and Plone REST API.

**Comment by rajatverma311201** [comment]
I want to do this task
Could you please tell me how to run this project and make changes
do i have to run make command only ?
or where can i get the guidelines

**Comment by stevepiercy** [comment]
@rajatverma311201 https://6.docs.plone.org/contributing/documentation/

**Comment by boss6825** [comment]
@rajatverma311201 are you still working on this?

**Comment by stevepiercy** [comment]
@boss6825 I unassigned @rajatverma311201, as they never submitted a pull request or pushed any commits for this issue.

**Comment by boss6825** [comment]
Thanks @stevepiercy 
I guess I will be working on this ...
I hope this is not a very complex issue

**Comment by stevepiercy** [comment]
@boss6825 it's relatively easy. I added that label just now. Feel free to ask questions for clarification of what is desired or if you get stuck.

**Comment by boss6825** [comment]
sorry to ask such a dumb question - I have been looking for the styling files in this documentation repo
and i was not able to find them...
Can you guide me with that?

**Comment by stevepiercy** [comment]
@boss6825 there are none in this repo, but any styles are inherited from the upstream themes, in order:

- https://github.com/plone/plone-sphinx-theme
- [sphinx-book-theme](https://sphinx-book-theme.readthedocs.io/en/latest/)
- [pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/en/stable/)

But modifying the CSS should not be necessary. Instead you can use https://sphinx-design.readthedocs.io/en/latest/ as noted in the description to accomplish the basic layout and design needed.

For icons, you could either use what Sphinx Design provides or reuse the Pastanaga set from Volto, for starters https://pastanaga.io/icons/.

**Comment by boss6825** [comment]
so should i fork this repo "https://github.com/plone/plone-sphinx-theme" and work in it to submit a linked PR to the plone-sphinx-theme repo

**Comment by stevepiercy** [comment]
> so should i fork this repo "https://github.com/plone/plone-sphinx-theme" and work in it to submit a linked PR to the plone-sphinx-theme repo

No. That repo merely provides the theme. You should not need to modify any CSS or the theme to accomplish this task. Please review the description again for direction.

**Comment by boss6825** [comment]
## Setting Up the Sphinx Theme for Plone Documentation

I was configuring the Sphinx theme for the documentation, following the [Plone Sphinx Documentation Guide](https://plone-sphinx-theme.readthedocs.io/guides/usage.html).  
Here’s a screenshot for reference:  
![Screenshot of the documentation setup](https://github.com/user-attachments/assets/39ebf094-edf4-4cbd-b589-750633fe52c9)

When I executed the command:  `pip install my_project[docs]`

I am getting this error about the caipyrinha package

![image](https://github.com/user-attachments/assets/50ef7e34-56ab-4fd3-bfc3-cf04ca079e6e)

I think this is because of python version 3.x compatibility issue with caipyrinha
I tried to use the latest version of caipyrinha from its repo but facing the same error

possible solution I can think of is - removing caipyrinha from pyproject.toml

can you provide some insight on this @stevepiercy

**Comment by stevepiercy** [comment]
> I was configuring the Sphinx theme for the documentation, following the [Plone Sphinx Documentation Guide](https://plone-sphinx-theme.readthedocs.io/guides/usage.html).

That's the wrong repo, as I said previously:

> No. That repo merely provides the theme.

The issue is in `plone/documentation`. See [Contribute to documentation](https://6.docs.plone.org/contributing/documentation/). Also, FYI, projects usually have a README or other file that says how to contribute, including this one https://github.com/plone/documentation?tab=readme-ov-file#contribute.

**Comment by Manas-Kenge** [comment]
Hey @boss6825 , I raised a pr for this issue as it was inactive for more than a month. I hope you don't mind
